pip/basecommand.py

```diff
@@ -64,6 +66,14 @@ def __init__(self, isolated=False):
         )
         self.parser.add_option_group(gen_opts)
 
+        if self.warn_outside_virtualenv:
+            self.cmd_opts.add_option(
+                '--force-use-system-python',
+                dest='force_use_system_python',
+                action='store_true',
+                help="Force command to use system python (work outside virtualenv)"
```

Comment:
I think this text is a bit misleading. Passing `--force-use-system-python` does not force system python, it allows system python.